api/ErrorEvent.json

```diff
@@ -21,7 +21,7 @@
             "version_added": true
           },
           "ie": {
-            "version_added": true
+            "version_added": "10"
```

Comment:
This is the event interface for "error" events fired at the window object for errors in script parsing/execution. https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror#browser_compatibility says that `onerror` was supported since IE9, so this needs double checking.

Reply:
`onerror` seems to simply return a string, not an `ErrorEvent` instance, so I'm not sure how to test this one...

Reply:
That string is probably the event handler code serialized. When the event is actually fired, what does the event object look like?

Making a working demo in a modern browser might be a good first step, so you're sure the problem is with IE if it still doesn't work.

Reply:
Using the same test code as in #10844 (AKA, `addEventListener`), I've determined that the error event handler throws a regular event in IE 9, but starts throwing an `ErrorEvent` in IE 10, so this checks out!

Reply:
Excellent, thanks for testing!